### PR TITLE
add delete image functionality

### PR DESCRIPTION
**Pull request by dotte-osu**
I realized that the page image files are not removed from our server when a pet or shelter is removed.

### CODE REVIEW

File: src/controllers/imageController.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 const multer = require('multer')
-const path = require('path');
```
Comment (dotte-osu) [inline src/controllers/imageController.js:2]: not used

File: src/controllers/petsController.js [hunk 2]
```diff
@@ -21,7 +23,7 @@ const getLatLngByZipCode = async (zipCode) => {
 };
 
 const getBatchLatLng = async (list) => {
-  return (await geocoder.batchGeocode(list.map(address => { return { q: address, country: 'US' }})))
+  return (await geocoder.batchGeocode(list.map(address => { return { q: address, country: 'US' } })))
```
Comment (dotte-osu) [inline src/controllers/petsController.js:26]: formatted the file

File: src/controllers/petsController.js [hunk 3]
```diff
@@ -168,11 +170,27 @@ const getPet = async (req, res, next) => {
...
 const deletePet = async (req, res, next) => {
   log.debug("Calling deletePet...");
+
+  //get image path
+  await getPetById(req.app.get('db'), req.params.petID)
+    .then((dbResponse) => {
+      if (dbResponse.length == 0) {
+        return res.status(404).json({ error: "Pet not found." });
+      }
+      removeImageFile(dbResponse[0].Picture)
```
Comment (asching7108) [inline src/controllers/petsController.js]: If we want to be extra careful, I would suggest moving the `deletePetById` part before `removeImageFile(dbResponse[0].Picture)`, to make sure we only delete the image after we've successfully deleted a pet.

**Review by asching7108 — COMMENTED** [review]
All looks good! Also thank you for fixing the indentation. I just have a suggestion.

File: src/controllers/petsController.js (hunk 3 above)
Comment (dotte-osu) [reply src/controllers/petsController.js]: thank you! fixed!

**Comment by dotte-osu** [comment]
@asching7108 Fixed and its ready for review!

**Review by xHoudek — APPROVED** [review]

**Pull request merged**